src/utils/getFileIconFromFileType.ts

```diff
@@ -0,0 +1,24 @@
+import { FileType, Icon } from '../tree/types';
+
+/** 1-to-1 maps a given file type to an Icon type, returning an emoji that
+ * corresponds to the specific file type
+ * @param {FileType} fileType - a type a file such as FileType.FILE, FileType.FOLDER,
+ * or FileType.CONFIG_FILE
+ * @returns {Icon} - an emoji passed into a string depending on which file
+ * type was found (FILE, CONFIG_FILE, or FOLDER)
+ */
+
+export const getFileIconFromFileType = (fileType: FileType): Icon => {
+  if (fileType === FileType.CONFIG_FILE) {
```

Comment:
Use something like Map[FileTypePassedin]
That way you don’t need an if statement for each map....

Reply:
Added that instead of return statement (Icon[fileType])